python/servo/visual_studio.py

```diff
@@ -145,9 +145,9 @@ def find_msvc_redist_dirs(vs_platform: str) -> Generator[str, None, None]:
                 path1 = os.path.join(vs_platform, "Microsoft.{}.CRT".format(redist_version))
                 path2 = os.path.join("onecore", vs_platform, "Microsoft.{}.CRT".format(redist_version))
                 for path in [path1, path2]:
-                    path = os.path.join(redist_path, path)
+                    full_path = os.path.join(redist_path, path)
                     if os.path.isdir(path):
```

Comment:
```suggestion
                    if os.path.isdir(full_path):
```
The code changes the variable name, so it has to change all the places where the original name was used